### DISCUSSION TITLE
Beginning documentation missing

### DISCUSSION THREAD

**Discussion by jidanni** [i3/i3#4708] [Q&A]
OK, I found
/usr/share/doc/i3-wm/userguide.html
but it should start from the very beginning
"After you do
apt install i3
then in you ~/.xsession you need to put..."

Also https://i3wm.org/docs/ doesn't mention it either.

**Comment by Airblader** [comment]
Can you elaborate more what it is you're missing? For example, I've used i3 for some 6+ years now and never once needed to touch `~/.xsession`. I don't think we should try to document how X works in various kinds of setup – that's a big task, and not ours.

**Comment by jidanni** [comment]
OK, so in my
.profile
I put
i3& ?
